libs/vendored/jwtool/src/lib/jwtool.ts

```diff
@@ -0,0 +1,211 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a TS rewrite of https://github.com/mozilla/fxa-jwtool/blob/master/index.js